content/en/docs/usage/kubectl-plugin.md

```diff
@@ -154,3 +159,50 @@ Approved CertificateRequest 'istio-system/mesh-ca'
 $ kubectl cert-manager deny -n my-app my-app --reason "example.com" --message "violates policy"
 Denied CertificateRequest 'my-app/my-app'
 ```
+
+---
+
+### Experimental
+The cert-manager plugin contains experimental commands for operations which are
```

Comment:
Nit- so that's it's obvious this doesn't talk about plugin as a whole
```suggestion
`kubectl cert-manager x` has experimental sub-commands for operations which are
```